modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/search/helper/DDMSearchUtil.java

```diff
@@ -153,7 +157,7 @@ public SearchContext buildTemplateSearchContext(
 		try {
 			searchContext.setAttribute(
 				"resourcePermissionName",
-				_ddmPermissionSupport.getTemplateModelResourceName(
+				ddmPermissionSupport.getTemplateModelResourceName(
```

Comment:
Hey @MrJohn1911, just pass the result of the `ddmPermissionSupport. getTemplateModelResourceName(resourceClassNameId)` if possible. Thx